libc/config/linux/arm/entrypoints.txt

```diff
@@ -367,6 +367,18 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.ufromfpxl
 )
 
+if(LLVM_LIBC_FULL_BUILD)
+  list(APPEND TARGET_LIBC_ENTRYPOINTS
+    # stdlib.h entrypoints
+    libc.src.stdlib._Exit
+    libc.src.stdlib.atexit
+    libc.src.stdlib.exit
```

Comment:
Trying to cross compile the unit tests is super broken, but very quickly, we will need realloc.